stylesheets/commons/Colours.less

```diff
@@ -676,6 +676,15 @@ ul.nav-tabs li.game-wiiu,
 	color: var( --clr-california-80, #f9dec7 );
 }
 
+/* Deadlock */
+.deadlock-amberhand-text {
+	color: #c09b54;
```

Comment:
sure that this color is suitable for both dark and light mode?
or do we need to define values for both modes?